pkg/providers/vsphere/testdata/expected_results_bottlerocket_mirror_config_cp.yaml

```diff
@@ -473,7 +473,7 @@ spec:
       datastore: /SDDC-Datacenter/datastore/WorkloadDatastore
       diskGiB: 25
       folder: '/SDDC-Datacenter/vm'
-      memoryMiB: 8192
+      memoryMiB: 4096
```

Comment:
I believe these changes are because the new code fixed a latent bug
I checked the input and I believe this is the correct output. Pls take a look and let me know if I'm mistaken
@abhinavmpandey08 @pokearu

Reply:
Done